tasks.py

```diff
@@ -853,7 +852,6 @@ class TestDependencies(NamedTuple):
             "reqs/requirements-dev-redshift.txt",
             "reqs/requirements-dev-snowflake.txt",
             # "Deprecated API features detected" warning/error for test_docs[split_data_on_whole_table_bigquery] when pandas>=2.0  # noqa: E501
-            "reqs/requirements-dev-sqlalchemy1.txt",
```

Comment:
Removing this requirement doesn't bump the SQLA version since `redshift-sqlalchemy` requires version `sqlalchemy-1.4.54`